lib/hexpm_web/templates/docs/public_keys.html.md

```diff
@@ -10,6 +10,8 @@ If you have been linked to this page from an error message, it means the public
 
 #### Active keys
 
+<!-- NOTE: When adding new public key, remember to update https://repo.hex.pm/public_key -->
+
```

Comment:
ideally this would have been `<% #NOTE: ... %>` but we don't support eex here. Even though we render this comment in the output html I think that's probably ok.